query-service-client/build.gradle.kts

```diff
@@ -12,5 +12,5 @@ dependencies {
   // Logging
   implementation("org.slf4j:slf4j-api:1.7.30")
   // Config
-  implementation("com.typesafe:config:1.3.2")
```

Comment:
while you are can you upgrade `io.grpc` dependencies to `1.31.1` and grpc utils to `0.1.4` and then get rid of the snyk ignore message? Keep the .snyk file but remove the vulnerability info. See https://github.com/hypertrace/java-grpc-utils/pull/4

Reply:
Done.